.github/CONTRIBUTING.md

```diff
@@ -134,9 +134,10 @@ Possible log levels are:`trace`, `debug`, `info`, `warn`, `err`, `critical`, `of
 * Go to `src/autopas/options/TuningStrategyOption.h`.
   * Add a new enum in `TuningStrategyOption::Value`.
   * Add a new string representation in the `map` of `TuningStrategyOption::getOptionNames()`.
-* In `src/autopas/AutoPas.h::generateTuningStrategy()`:
+* In `src/autopas/selectors/tuningStrategy/TuningStrategyFactory.cpp::generateTuningStrategy()`:
   * Add a `case` for the new strategy.
-  * If the new strategy handles communication between processes itself, make sure to not wrap it in the lower `mpiStrategyOption`-switch.
+  * If the new strategy handles communication between processes itself, make sure to not wrap it in the
+    lower `mpiStrategyOption`-switch.
```

Comment:
```suggestion
  * If the new strategy handles communication between processes itself, make sure to not wrap it in the lower `mpiStrategyOption`-switch.
```
The rest of the file also leaves the line wrapping to the viewer, hence I would keep this for consistency